### ISSUE TITLE
import DCN failed

### ISSUE THREAD

**Issue by xiao1512** [yichen928/SparseFusion#51]
请问有没有人遇到过“import DCN failed”这个问题呀？似乎是可以正常训练的，目前处在这个阶段。
2025-09-10 09:50:48,709 - mmdet - INFO - workflow: [('train', 1)], max: 6 epochs
谢谢各位大佬

**Comment by yichen928** [comment]
You can just ignore it. We do not use DCN.

**Comment by xiao1512** [comment]
> You can just ignore it. We do not use DCN.

thank you.

**Comment by xiao1512** [comment]
> You can just ignore it. We do not use DCN.


I would like to ask a basic question. Since I only have two A6000s, I plan to set the batch_size to 8. Do I need to cut the learning rate in half?

**Comment by xiao1512** [comment]
> You can just ignore it. We do not use DCN.


Because I saw a comment in /home/ubuntu/WorkSpace/xjf/SparseFusion-main/configs/sparsefusion_nusc_voxel_LC_r50.py: # our default setting uses 4 GPUs with 4 samples per-GPU, please ensure the LR consistent with your batch size.
